acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/dsl/tls/OkHttpClientHelpers.java

```diff
@@ -48,8 +48,11 @@
 
   public static OkHttpClient createOkHttpClient(
       final Optional<ClientTlsConfig> clientTlsConfiguration) {
+
+    final int secondsToWait = Boolean.getBoolean("debugSubProcess") ? 3600 : 10;
+
     final OkHttpClient.Builder clientBuilder =
-        new OkHttpClient.Builder().readTimeout(Duration.ofSeconds(10));
+        new OkHttpClient.Builder().readTimeout(Duration.ofSeconds(secondsToWait));
```

Comment:
`readTimeout` is set when `debugSubProcess` property exists, is that naming deliberate?

Reply:
Yes - i.e. if we're debugging the EthSIgner Subprocess, we need more than 10 seconds to step through the EthSigner code to determine what's happening before the connection is dropped.